types/src/v0/impls/header.rs

```diff
@@ -592,6 +604,7 @@ impl ExplorerHeader<SeqTypes> for Header {
     type ProposerId = FeeAccount;
     type NamespaceId = NamespaceId;
 
+    // TODO what are these expected values w/ multiple Fees
```

Comment:
How do we determine `proposer_id` w/ multiple Fees? Can this be resolved in block explorer by propagating the `Vec` upward?

Reply:
@Ayiga for your reference.

Reply:
What is the proposer id currently?  Is it the HotShot leader id or the builder id?

Reply:
Here they are taking the account field of `FeeInfo` (builder account) for use as proposer id. Which I think makes sense in the case of the single fee (since the since that is the account paying the fee). With multiple fees not sure how this would work.